### DISCUSSION TITLE
ILI9481

### DISCUSSION THREAD

**Discussion by salarizadi** [Moddable-OpenSource/moddable#1093] [Q&A] [answered]
Hi. 

Does the ILI9341 driver module for the display work with the ILI9481 driver?

**Comment by phoddie** [comment, marked as answer]
Yes, the ILI9341 driver module implements the MIPI display commands over SPI. It works with many different display modules from different manufacturers, for example the ST7789, that support the same command set and transport. The ILI9481 supports bigger screens than  ILI9341 (320x480 v. 240x320), but is otherwise the same. With the appropriate configuration for your LCD panel, it should work with ILI9481.

The [documentation for the Moddable SDK's ILI9341 driver](https://github.com/Moddable-OpenSource/moddable/blob/public/documentation/drivers/ili9341/ili9341.md) covers more details about capabilities and configuration.
